test/astar.cc

```diff
@@ -956,7 +956,7 @@ void test_backtrack_complex_restriction(int date_time_type) {
     default:
       throw std::runtime_error("unhandled case");
   }
-  EXPECT_EQ(leg.shape(), correct_shape)
+  EXPECT_TRUE(test::encoded_shape_equality(leg.shape(), correct_shape))
```

Comment:
all the shape related failures look like th is now calling this helper function which does hausdorff distance to compare how close in distance the shapes are to each other